internal/services/interface.go

```diff
@@ -46,6 +47,17 @@ import (
 	upgradeservice "github.com/juju/juju/domain/upgrade/service"
 )
 
+// ModelService provides access to the model services required by the
+// apiserver.
+type ModelService interface {
```

Comment:
This doesn't belong here. Move it to the `modelworkermanager` package, next to the `MetricsSink` interface

Reply:
Got it, thanks Ian :)